infrastructure/keyvault.tf

```diff
@@ -0,0 +1,29 @@
+resource "azurerm_key_vault" "kv" {
```

Comment:
should probably use the module we have for it?

https://github.com/hmcts/cnp-module-key-vault